Did you mean taxi data instead of tweets?

lenadroid opened the pull request:
Great example! noticed that "tweets" are being used in a few headers. Not sure if this is intentional. Feel free to dismiss if it was.

azurecla commented:
Hi __@rgardler__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

lenadroid committed:
Did you mean taxi data instead of tweets?

Great example! noticed that "tweets" are being used in a few headers. Not sure if this is intentional. Feel free to dismiss if it was.
